eng/common/scripts/CadlSyncAndGenerateScripts.md

```diff
@@ -0,0 +1,127 @@
+# Integrating with cadl sync and generate scripts
```

Comment:
Just so this file shows up next to the scripts I would rename it to exactly match one of the scripts with a .md at the end, ex Cadl-Project-Generate.ps1.md. Or if you want it to cover both I would still call make it match the prefix so perhaps Cadl-Project-Scripts.md